lib/CacheMapper.ts

```diff
@@ -0,0 +1,163 @@
+export class CacheMapper<K, V> implements Map<K, V> {
```

Comment:
Can you make `V` to `any` by default? This would be helpful for the existing js code migration. We can always explicit define `T` anyway.

Reply:
yeah that was same as `T` because any generic type can be assigned to any `words without spaces`

Reply:
i create `K` for KEY and `V` for VALUE so can be easily undestand what that variable means

Reply:
and any generic constant are treat as `any` by default

Reply:
sorry for my bad english :)

Reply:
and we no need using `CacheMapper` class (we must hide this cache mapper from index export), we just using the `Cache<T>` class

Reply:
CacheMapper and Cache generic type fixed and supported to integrated with `hexo.locals` check Test CI https://github.com/dimaslanjaka/hexo/actions/runs/4990595464/jobs/8936162528